code/modules/vehicles/wheelchair.dm

```diff
@@ -13,6 +13,8 @@
 	var/delay_multiplier = 6.7
 	/// This variable is used to specify which overlay icon is used for the wheelchair, ensures wheelchair can cover your legs
 	var/overlay_icon = "wheelchair_overlay"
+	var/foldabletype = /obj/item/wheelchair //Determines what the object folds into
+	var/can_fold = TRUE //Determines that the object can fold up into something
```

Comment:
```suggestion
	/// Determines what the object folds into.
	var/foldabletype = /obj/item/wheelchair
	/// Determines that the object can fold up into something.
	var/can_fold = TRUE
```
Document new variables using the dmdoc format, like this: https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md